src/elements/category.js

```diff
@@ -0,0 +1,40 @@
+export default function(namespace) {
+  const ArrayElement = namespace.getElementClass('array');
+
+  class Category extends ArrayElement {
+    constructor() {
+      super(...arguments);
+      this.element = 'category';
+    }
+
+    get resourceGroups() {
+      return this.children((item) => item.classes.contains('resourceGroup'));
+    }
+
+    get dataStructures() {
+      return this.children((item) => item.classes.contains('dataStructures'));
+    }
+
+    get scenarios() {
+      return this.children((item) => item.classes.contains('scenario'));
+    }
+
+    get transitionGroups() {
+      return this.children((item) => item.classes.contains('transitions'));
+    }
+
+    get resources() {
+      return this.children((item) => item.element === 'resource');
+    }
+
+    get transitions() {
+      return this.children((item) => item.element === 'transition');
+    }
+
+    get copy() {
```

Comment:
These methods are plural because they return an array. Copy is a special case, because in journalism the term "copy" refers to all the sections of text in the body of a work (or even multiple works) that a copy editor would work with, so I *think* (but am not 100% sure) that the correct term here is still just "copy" instead of "copies." You can think of it like water - a measurable amount but never called "waters." Anybody else have thoughts or insight on this?

http://english.stackexchange.com/questions/9941/what-is-the-plural-of-copy-when-applied-to-the-output-of-copywriters